btn-1cc-checkout.js

```diff
@@ -43,6 +43,56 @@ function btnCheckout(){
     });
   }
 
+  // For attaching event listener to Woodmart's sticky add-to-cart
+  document.addEventListener('scroll',(e)=>{
+     
+    var stickyBtn=document.querySelectorAll('#btn-1cc-pdp')[1];
+
+    if (stickyBtn != null) {
+        stickyBtn.onclick = function() {
```

Comment:
Can we make it a common function and invoke it in both places? @angbpy

Reply:
@ChetanGN Done!